examples/pong_tutorial_02/pong.rs

```diff
@@ -15,15 +15,13 @@ const PADDLE_WIDTH: f32 = 4.0;
 pub struct Pong;
 
 impl SimpleState for Pong {
-    fn on_start(&mut self, data: StateData<'_, GameData<'_, '_>>) {
+    fn on_start(&mut self, data: StateData<'_, GameData>) {
         let world = data.world;
 
         // Load the spritesheet necessary to render the graphics.
         // `spritesheet` is the layout of the sprites on the image;
         // `texture` is the pixel data.
-        let sprite_sheet_handle = load_sprite_sheet(world);
-
-        world.register::<Paddle>();
```

Comment:
registering Paddle was needed for specs by design, right?

Reply:
yes, in an ecstatics like specs, you register your components so that it knows how to store them.
in an ecstatics like legion, based on archetypes, components storage directly depends on how you add them to entities